README.md

```diff
@@ -42,3 +43,48 @@ Sample Request
 
 **NOTE:** 
 Remember to add your timezone to the DateTimeFormat.ISO
+
+### Health Endpoint
+The health endpoint of the opensrp server is `/opensrp/health`. It always returns information in json format. The status code of the response can either be `200` or `500` depending on status of the services. Response status code is `200` if all the services are running ok but `500` if any service is down/inaccessible.
```

Comment:
Could we have used a status [503](https://httpstatuses.com/503) - Service Unavailable - since one of the services we depend on is unavailable? And leave status [500](https://httpstatuses.com/500) -  Internal Server Error - which could signal a code or configuration issue?

Reply:
yeah updated it.